plugins/woocommerce-admin/client/marketplace/components/category-selector/category-dropdown.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { Dropdown } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { Category } from './category-selector';
+
+function DropdownContent( props: {
+	readonly categories: Category[];
+} ): JSX.Element {
+	return (
+		<ul className="woo-marketplace__category-dropdown-list">
```

Comment:
We probably need to make a final decision on if we're using the `woo-marketplace__` or `woocommerce-marketplace__` namespace for styles. I like this short version, although we've ended up using `woocommerce-marketplace__` everywhere else I believe. 

Not blocking, we should just update this everywhere later on once we decide.

Reply:
Looking at classnames here and there in Woo Core, I found examples like this:

```
woocommerce-dashboard__chart-block
woocommerce-embedded-layout__primary
woocommerce-inbox-message-exit-active
woocommerce-layout__footer
woocommerce-marketing-recommended-extensions-item
woocommerce-review-activity-card__verified
woocommerce-table__download-button
```

I also like the shorter `woo-marketplace`, but I'd say that for consistency with classes elsewhere, we should carry on with `woocommerce-marketplace__`.

Reply:
OK, I'm happy to change all of these to `woocommerce-marketplace`. Thanks for the heads up!